product-form/form.js

```diff
@@ -23,4 +25,67 @@ addForm.addEventListener('submit', (e) => {
         price: Number(price)
     };
     addProduct(newProduct);
-});
\ No newline at end of file
+    location.reload();
+});
+
+const ul = document.querySelector('#removeList');
+
+seedAndGetProducts();
+
+const coffees = getFromLocalStorage('PRODUCTS');
+
+for (let i = 0; i < coffees.length; i++){
+    const coffee = coffees[i];
+    const li = renderDelete(coffee);
+    ul.appendChild(li);
+}
+
+export function renderDelete(coffee){
+
+    const li = document.createElement('li');
+    li.classList.add('product');
+
+    const h2 = document.createElement('h2');
+    h2.textContent = coffee.name;
+
+    const img = document.createElement('img');
+    img.src = `../assets/${coffee.image}`;
+
+    const buttonDelete = document.createElement('button');
+    buttonDelete.id = coffee.id;
+    buttonDelete.textContent = 'Remove';
+
+
+    renderRemoveListener(buttonDelete, coffee.id);
+
+    li.append(h2, img, buttonDelete);
+
+    return li;
+}
+
+function renderRemoveListener(buttonDelete, id){
```

Comment:
nice work hitting that stretch goal! Your site is really user friendly, and this remove button functions very well.